templates/maldi-pipeline.ipynb

```diff
@@ -336,9 +336,7 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    # cores_to_crop = region_core_info["Core"].unique().tolist()
</pre><p>We can make this a bit easier for the user, maybe inside the <code>extraction.generate_glycan_mask</code> function allow <code>cores_to_crop</code> to optionally be <code>"all"</code> and we can move this complexity into the function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/maldi-tools/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>